changelog/7527.improvement.rst

```diff
@@ -0,0 +1 @@
+Field name is now shown in the diff message for namedtuple instead of index.
```

Comment:
```suggestion
When a comparison between `namedtuple` instances fail, pytest now shows the differing field names instead of their indexes.
```

Reply:
Thanks, changed to this with a couple of adjustments.